package.json

```diff
@@ -1,6 +1,19 @@
 {
   "name": "homeday-blocks",
   "version": "2.5.0",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/homeday-de/homeday-blocks.git"
+  },
+  "keywords": [
+    "vue.js",
```

Comment:
If you have a better suggestion here, please shoot. :)

Reply:
just `vue` as the Vue repo also uses it

![image](https://user-images.githubusercontent.com/1734873/67286270-6587f400-f4d9-11e9-9432-08349a59e7b4.png)

Reply:
Good reference. :+1: